fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/impl/object/builder/VillagerTaskProviderInternals.java

```diff
@@ -0,0 +1,116 @@
+package net.fabricmc.fabric.impl.object.builder;
+
+import com.google.common.collect.ImmutableList;
+import com.mojang.datafixers.util.Pair;
+import net.fabricmc.fabric.api.object.builder.v1.ai.tasks.*;
+import net.minecraft.entity.ai.brain.task.Task;
+import net.minecraft.entity.passive.VillagerEntity;
+import net.minecraft.village.VillagerProfession;
+//import org.jetbrains.annotations.Nullable;
+
+import java.util.Map;
+
+import static net.fabricmc.fabric.api.object.builder.v1.ai.tasks.VillagerTaskProvider.TaskType.PLAY;
+
+
+public class VillagerTaskProviderInternals {
+	
+	private static final Map<VillagerProfession, VillagerTaskProvider> villagerTaskProviderMap;
+	
+	private static final VillagerTaskProvider defaultTaskProvider = new VillagerTaskProvider();
+	
+	private static VillagerTaskProvider getTaskProvider(VillagerProfession profession)
+	{
+		assert(profession != null);
+		
+		if (profession == VillagerProfession.NONE)
```

Comment:
Also some checkstyle issues. Get the Checkstyle plugin and ensure everything is all right.